internal/exeutils/openuri.go

```diff
@@ -0,0 +1,15 @@
+// +build !windows,!darwin
```

Comment:
The `exeutils` is for dealing with executables. Opening browser URI's doesn't feel like it belongs here. Perhaps `osutils` would be more appropriate, seeing as it's a very OS specific thing.

Reply:
moved